data-model/src/main/java/io/micronaut/data/model/query/builder/sql/SqlQueryBuilder.java

```diff
@@ -1618,331 +1627,6 @@ protected GeneratedValue.Type selectAutoStrategy(PersistentProperty property) {
         return GeneratedValue.Type.AUTO;
     }
 
-    private String addTypeToColumn(PersistentProperty prop, String column, boolean required) {
```

Comment:
I had to extract this to the util class since checkstyle was compaining on long lines (2 classes now and was failing)